proposals/0035-linalg-matrix.md

```diff
@@ -1210,6 +1239,27 @@ Validation rules will enforce that:
 
 Must be called from wave-uniform control flow.
 
+
+```llvm
+declare void @dx.op.matrixBinOp(
```

Comment:
Not a strong opinion but I think we talked about this before but this isn't really a BinOp right? Since its not taking two values and producing a third instead it's taking two values and then assigning one of them.

If we only expect to have the one operation maybe it could be the more direct `matrixAccumulate`?